test/e2e/node/pods.go

```diff
@@ -229,7 +229,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 			)
 
 			const delay = 2000
-			const workers = 3
+			const workers = 2
```

Comment:
this is not the same as nodes... this is the number of worker threads

Reply:
:facepalm: